ZenPacks/zenoss/OpenStackInfrastructure/datasources/EventsAMQPDataSource.py

```diff
@@ -29,12 +29,15 @@
 
 from ZenPacks.zenoss.OpenStackInfrastructure.utils import ExpiringFIFO, amqp_timestamp_to_int
 from ZenPacks.zenoss.OpenStackInfrastructure.events import event_is_mapped, map_event
+from ZenPacks.zenoss.OpenStackInfrastructure.datamaps import ConsolidatingObjectMapQueue
 
 # How long to cache data in memory before discarding it (data that
 # is coming from ceilometer, but not consumed by any monitoring templates).
 # Should be at least the cycle interval.
 CACHE_EXPIRE_TIME = 15 * 60
 
+MAP_QUEUE = defaultdict(ConsolidatingObjectMapQueue)
```

Comment:
This provides no way to change the various timeouts

Reply:
Good point- i've added some zProperties to address this.